repo-agent/images/issue-sandbox/Dockerfile

```diff
@@ -1,5 +1,21 @@
-# Build stage
+# Build the issue-sandbox binary
+FROM golang:1.24.7 as builder
+
+WORKDIR /workspace
+# Copy the Go Modules manifests
+COPY go.mod go.mod
+COPY go.sum go.sum
+# cache deps before building and copying source so that we don't need to re-download as much
+# and so that source changes don't invalidate our downloaded layer
+RUN go mod download
+
+# Copy the go source
+COPY issue-sandbox/main.go issue-sandbox/main.go
+
+# Build
+RUN CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a -o issue-sandbox issue-sandbox/main.go
+
+# Envbuilder root
 FROM ghcr.io/coder/envbuilder
-ENV KANIKO_DIR=/.envbuilder
-# injecting devcontainer.json via `devcontainerConfigRef` in RGD
-COPY issue-sandbox/gemini-cli-entrypoint.sh /issue-agent/gemini-cli-entrypoint.sh
\ No newline at end of file
+COPY --from=builder /workspace/issue-sandbox /repo-agent/issue-sandbox
+ENV KANIKO_DIR=/.envbuilder
```

Comment:
Please add a newline at the end of the file for consistency with Dockerfile best practices.